file-formats/sito/examples/TM_SITO_TRANS_TEST_01.sito.json

```
@@ -0,0 +1,410 @@
+        "sitnactiontype": "02",
+        "sitnobjactscope": "01",
+        "sitnactionid": "SITDEMO_ACT_REBOOK_BOOKING",
+        "sit2ObjActT": [
+          {
+            "language": "en",
+            "sitnactionname": "Rebook Passengers",
+            "sitnactiondescription": "Change bookings to the next flight with the same airline and connection ID"
+          },
+          {
+            "language": "de",
+            "sitnactionname": "Passagiere umbuchen",
+            "sitnactiondescription": "Buchungen auf den nächten Flug mit der gleichen Fluggesellschaft und Verbindungs-ID ändern"
+          }
+        ],
+        "sit2ObjActCb": [
+          {
+            "sitncallbackid": "SITDEMO_CB_REBOOK_BOOKING"
+          }
+        ]
+      },
+      {
+        "sitnactiontype": "02",
+        "sitnobjactscope": "01",
+        "sitnactionid": "SITDEMO_ACT_DELETE_BOOKING",
+        "sit2ObjActT": [
+          {
+            "language": "en",
+            "sitnactionname": "Cancel Bookings",
+            "sitnactiondescription": "Delete the bookings from the flight"
+          },
+          {
+            "language": "de",
+            "sitnactionname": "Buchungen stornieren",
+            "sitnactiondescription": "Buchungen aus dem Flug löschen"
+          }
+        ],
+        "sit2ObjActCb": [
+          {
+            "sitncallbackid": "SITDEMO_CB_DELETE_BOOKING"
+          }
+        ]
+      },
+      {
+        "sitnactiontype": "01",
+        "sitnobjactscope": "01",
+        "sitnactionid": "SITDEMO_ACT_MANAGE_FLIGHT",
+        "sit2ObjActT": [
+          {
+            "language": "en",
+            "sitnactionname": "Manage Flight Bookings",
+            "sitnactiondescription": "Maintain booking details of the flight"
+          },
+          {
+            "language": "de",
+            "sitnactionname": "Flugbuchungen verwalten",
+            "sitnactiondescription": "Buchungsdetails des Flugs pflegen"
+          }
+        ],
+        "sit2ObjActNav": [
+          {
+            "sitnnavigationid": "SITDEMO_NAV_MANAGE_FLIGHT"
+          }
+        ]
+      }
+    ],
+    "sit2ObjCb": [
+      {
+        "sitnobjclbkscope": "01",
+        "sitncallbackclassname": "CL_SIT2_DEMO_CB_CANCEL_BOOKING",
+        "sitncallbackid": "SITDEMO_CB_DELETE_BOOKING",
+        "sit2ObjCbPar": [
+          {
+            "sitncallbackparamname": "BOOKID"
+          },
+          {
+            "sitncallbackparamname": "CARRID"
+          },
+          {
+            "sitncallbackparamname": "CONNID"
+          },
+          {
+            "sitncallbackparamname": "FLDATE"
+          }
+        ]
+      },
+      {
+        "sitnobjclbkscope": "01",
+        "sitncallbackclassname": "CL_SIT2_DEMO_CB_REBOOK_BOOKING",
+        "sitncallbackid": "SITDEMO_CB_REBOOK_BOOKING",
+        "sit2ObjCbPar": [
+          {
+            "sitncallbackparamname": "BOOKID"
+          },
+          {
+            "sitncallbackparamname": "CARRID"
+          },
```

If you rename the fields you can even think of something like this:
```suggestion
        "callbackParameters": [
          {
            "name": "BOOKID"
          },
          {
            "name": "CARRID"
          },
```

or even if you don't need additional information for the parameters.
```suggestion
        "parameters": [ "BOOKID", "CARRID"

```